eventmesh-registry-plugin/eventmesh-registry-zookeeper/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+    implementation 'log4j:log4j:1.2.17'
+    implementation 'org.apache.zookeeper:zookeeper:3.4.6'
+    implementation 'org.apache.curator:curator-client:4.0.1'
+    implementation 'org.apache.curator:curator-framework:4.0.1'
+    implementation 'org.apache.curator:curator-recipes:4.0.1'
+    implementation project(":eventmesh-registry-plugin:eventmesh-registry-api")
+    implementation project(":eventmesh-common")
+    testImplementation 'org.mockito:mockito-core'
+    testImplementation 'org.apache.curator:curator-test:2.12.0'
```

Comment:
Please manage the dependency version in root/build.gradle, and I remember we already have log4j in our dependency.